docs/setup/upgrade/upgrade-migrations.asciidoc

```diff
@@ -58,6 +57,29 @@ complete the upgrade migration before bringing up the remaining instances.
 ==== Preventing migration failures
 Review the common causes of {kib} upgrade failures and how to prevent them.
 
+[float]
+====== Documents were found for unknown saved object types
+Migrations will fail if saved objects were found which belong to an unknown
+saved object type. This could be caused by writing malformed documents directly
+to the Elasticsearch index, or by disabling a plugin that had previously
+created a saved object.
+
+We recommend using the https://www.elastic.co/guide/en/kibana/7.17/upgrade-assistant.html[Upgrade Assistant] 
+to discover and remedy any unknown saved object types. In addition to the 
+upgrade assistant, {kib} version 7.17.0 deployments containing unknown saved
```

Comment:
Leading this sentence with "In addition to the Upgrade Assistant" confused me a bit.  I think you can remove that lead-in and simply use:

> {kib} version 7.17.0 deployments containing unknown saved object types will also log the following warning message.